File: Functions/Javascript/func.js
function say_hello() {
  console.log("Hello, World!");
}

function add(a, b) {
  return a + b;
}

const subtract = (a, b) => {
  return a - b;
};

function main() {
  say_hello();
  console.log(`5 + 2 = ${add(5, 2)}`);
  console.log("5 - 2 =", subtract(5, 2));
}

main();

File: Functions/Go/func.go
package main

import "fmt"

func sayHello() {
	fmt.Println("Hello, World!")
}

func add(a int, b int) int {
	return a + b
}

func main() {
	subtract := func(a int, b int) int {
		return a - b
	}

	var sum int = add(5, 2)
	var diff int = subtract(5, 2)
	
	sayHello()
	fmt.Println("5 + 2 = ", sum)
	fmt.Println("5 - 2 = ", diff)
}